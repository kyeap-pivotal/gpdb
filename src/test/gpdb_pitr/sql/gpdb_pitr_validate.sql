-- The DELETE was not replayed so should show all 10 rows.
SELECT * FROM gpdb_pitr_table;

-- The INSERT happened after the restore point was created
-- so this table should be empty.
SELECT * FROM gpdb_restore_points;
