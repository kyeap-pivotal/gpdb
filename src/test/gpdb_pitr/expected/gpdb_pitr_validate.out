-- The DELETE was not replayed so should show all 10 rows.
SELECT * FROM gpdb_pitr_table;
 num 
-----
   2
   3
   4
   7
   8
   5
   6
   9
  10
   1
(10 rows)

-- The INSERT happened after the restore point was created
-- so this table should be empty.
SELECT * FROM gpdb_restore_points;
 rpname | rplsn 
--------+-------
(0 rows)

